cmd/authctl/user/set-uid.go

```diff
@@ -0,0 +1,47 @@
+package user
+
+import (
+	"context"
+	"fmt"
+	"strconv"
+
+	"github.com/spf13/cobra"
+	"github.com/ubuntu/authd/internal/proto/authd"
+)
+
+// setUIDCmd is a command to set the UID of a user managed by authd.
+var setUIDCmd = &cobra.Command{
+	Use:   "set-uid <name> <uid>",
+	Short: "Set the UID of a user managed by authd",
+	Args:  cobra.ExactArgs(2),
+	RunE: func(cmd *cobra.Command, args []string) error {
+		name := args[0]
+		uidStr := args[1]
+		uid, err := strconv.ParseUint(uidStr, 10, 32)
+		if err != nil {
+			return err
+		}
+
+		fmt.Printf("Setting UID of user %q to %d...\n", name, uid)
+
+		client, err := NewUserServiceClient()
+		if err != nil {
+			return err
+		}
+
+		resp, err := client.SetUserID(context.Background(), &authd.SetUserIDRequest{
+			Name: name,
+			Id:   uint32(uid),
+		})
+		if err != nil {
+			return err
+		}
+
+		// Print any warnings returned by the server.
+		for _, warning := range resp.Warnings {
+			fmt.Printf("Warning: %s\n", warning)
```

Comment:
ditto

Reply:
Those are meant for the user. The warnings we currently use are:

https://github.com/ubuntu/authd/blob/59488fa1dc2a945f3f034b8d4eee0adf6c558f47/internal/users/manager.go#L407-L411

and 

https://github.com/ubuntu/authd/blob/59488fa1dc2a945f3f034b8d4eee0adf6c558f47/internal/users/manager.go#L415-L420